components/Form/form.css

```diff
@@ -0,0 +1,119 @@
+:root {
+  --primary-color: #ff6d6d;
+  --text-color: rgb(73, 73, 73);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.form_container {
+  margin: 1rem;
+}
+
+.form_main_container {
+  display: flex;
+  margin: 2rem 0rem;
+  flex-direction: column;
+}
+
+.form_label,
+.form_input {
+  color: var(--text-color);
+  padding: 1rem 0.5rem;
+}
+
+.form_input {
+  border-radius: 5px;
+  border: 1px solid rgb(179, 179, 179);
```

Comment:
as you have used var everywhere to maintain consistency you can make changes here.

Reply:
Okay